### PR TITLE
GDV-212 Feat timeseries wrong plot

### PR DESCRIPTION
**Pull request by CindyvdVries**
I made this branch from Feat-theming-and-toggle-raster.
Not up to date with the latest version of that branch.

### CODE REVIEW

File: components/graph-line.vue [hunk 1]
```diff
@@ -18,7 +18,11 @@
       {{ title }}
     </figcaption>
     <div v-if="!isCollapsed" class="graph-line__aspect-ratio">
-      <div v-echarts="{ data }" class="graph-line__chart" />
+      <div
+        :id="title"
+        v-echarts="{ data: graphData() }"
```
Comment (SiggyF) [inline components/graph-line.vue]: replace with [vue-echarts](https://github.com/ecomfe/vue-echarts)

File: layouts/default.vue [hunk 2]
```diff
@@ -156,6 +161,16 @@ export default {
       )
       this.updateRoute(newRouteObject)
     },
+    changeTheme() {
```
Comment (SiggyF) [inline layouts/default.vue]: comments?

File: store/map/index.js [hunk 3]
```diff
@@ -160,6 +172,12 @@ export const actions = {
 }
 
 export const getters = {
+  getThemes(state) {
```
Comment (SiggyF) [inline store/map/index.js]: why not in themes? Keep your store clean with [modules](https://vuex.vuejs.org/guide/modules.html). Or don't try to put everything in the store.

File: test/unit/store/map/index/actions.spec.js [hunk 4]
```diff
@@ -183,8 +202,8 @@ describe('loadPointDataForLocation', () => {
     }
     getFromApi.mockResolvedValue(apiResult)
     await actions.loadPointDataForLocation(
-      { commit },
-      { datasetIds: 'ab,cd', locationId: 'ef' },
+      { commit, state: {} },
+      { datasetIds: ['ab', 'cd'], locationId: 'ef' },
```
Comment (SiggyF) [inline test/unit/store/map/index/actions.spec.js]: please use recognizable ids

File: components/graph-line.vue (hunk 1 above)
Comment (CindyvdVries) [reply components/graph-line.vue]: implemented. Autoresize works properly on window update

File: store/map/index.js (hunk 3 above)
Comment (CindyvdVries) [reply store/map/index.js]: moved it to themes.js

File: test/unit/store/map/index/actions.spec.js (hunk 4 above)
Comment (CindyvdVries) [reply test/unit/store/map/index/actions.spec.js]: this is just a test

**Pull request merged**